src/NuGet.Clients/NuGet.CommandLine/Commands/DownloadCommandBase.cs

```diff
@@ -49,18 +52,19 @@ internal void CalculateEffectivePackageSaveMode()
                 packageSaveModeValue = SettingsUtility.GetConfigValue(Settings, "PackageSaveMode");
             }
 
-            EffectivePackageSaveMode = PackageSaveModes.None;
             if (!string.IsNullOrEmpty(packageSaveModeValue))
             {
+                // Restore \ Install always extract files
+                EffectivePackageSaveMode = Packaging.PackageSaveMode.Files;
```

Comment:
In another command (`OfflineFeedCommand`), we need to extract `nupkg` + `nuspec` by default; unless a flag `Expand` is set at which point it is `files + nupkg + nuspec`. 

I collated the two behaviors into a single flag rather than a third variable that represented this. I can change this back if it's problematic.